From b84a4bc8ec8f1d98ef5d6ed9ccca486bbaca4f4a Mon Sep 17 00:00:00 2001
From: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Date: Mon, 3 Dec 2018 15:04:08 +0200
Subject: [PATCH 5237/5242] MA-13638-1: Revert "MLK-20361: drm/panel/raydium:
 increase vsync_len to 4"

commit  e820bcc7b993cab60f757d052e6edfa203a57ca1 from
https://source.codeaurora.org/external/imx/linux-imx.git

This reverts commit 3c53532c466add234ac3ca2e70cefe80ea308b4f.

This change affects iMX8MM functionality, generating flicker.

Signed-off-by: Laurentiu Palcu <laurentiu.palcu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/panel/panel-raydium-rm67191.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/panel/panel-raydium-rm67191.c b/drivers/gpu/drm/panel/panel-raydium-rm67191.c
index 647ba3f..940f780 100644
--- a/drivers/gpu/drm/panel/panel-raydium-rm67191.c
+++ b/drivers/gpu/drm/panel/panel-raydium-rm67191.c
@@ -528,7 +528,7 @@ static int rad_bl_update_status(struct backlight_device *bl)
 	.hback_porch = { 34, 34, 34 },
 	.vactive = { 1920, 1920, 1920 },
 	.vfront_porch = { 10, 10, 10 },
-	.vsync_len = { 4, 4, 4 },
+	.vsync_len = { 2, 2, 2 },
 	.vback_porch = { 4, 4, 4 },
 	.flags = DISPLAY_FLAGS_HSYNC_LOW |
 		 DISPLAY_FLAGS_VSYNC_LOW |
-- 
1.7.9.5

